cms/djangoapps/contentstore/outlines.py

```diff
@@ -306,7 +327,18 @@ def get_outline_from_modulestore(course_key) -> Tuple[CourseOutlineData, List[Co
         # Pull course with depth=3 so we prefetch Section -> Sequence -> Unit
         course = store.get_course(course_key, depth=3)
         sections_data = []
-        for section in course.get_children():
+        sections = course.get_children()
+        for i, section in enumerate(sections):
+            for j in range(i + 1, len(sections)):
+                if i == j:
+                    break
+                section_1_children = section.children
+                section_2_children = sections[j].children
+                for index, child in enumerate(section_2_children):
+                    if child in section_1_children:
+                        acyclical_errors = _error_for_duplicate_child(sections[j], sections[j].get_children()[index])
+                        content_errors.append(acyclical_errors)
+                        section_2_children.remove(child)
```

Comment:
Also, note that we don't want to remove the child. We want to ignore/skip all but the first instance of this sequence for the purposes of pushing data into `learning_sequences`, but we don't want to delete it from the original course.

So maybe this checking needs to happen somewhere deeper, like something we pass into `_make_section_data`?

I guess the other thing to consider is that we could make a smaller data structure that has just the data we need from modulestore, and then filter/remove nodes from that. This approach might make more sense when we have to do things like add support for split_test when it's a child of a chapter.